Sources/CocoaLumberjack/include/CocoaLumberjack/DDLog.h

```diff
@@ -780,7 +780,7 @@ typedef NS_OPTIONS(NSInteger, DDLogMessageOptions){
     NSInteger _context;
     NSString *_file;
     NSString *_fileName;
-    NSString *_function;
+    NSString * _Nullable _function;
```

Comment:
Not a review comment but a remark for release: This is a breaking change.

Reply:
It should not break anything since `_Nullable` is just an annotation: it could only generates warnings.

Reply:
Gotcha! removing it.